src/sentry/static/sentry/app/views/settings/organization/members/organizationMembersView.jsx

```diff
@@ -232,23 +278,16 @@ class OrganizationMembersView extends AsyncView {
         />
 
         <Panel>
-          <PanelHeader disablePadding>
-            <Box px={2} flex="1">
-              {t('Member')}
-            </Box>
-            <Box px={2} w={180}>
-              <GuideAnchor target="member_status" type="text">
```

Comment:
@getsentry/growth Heads up these headers are going away, are they important enough to keep around?

Reply:
That's fine—I can re-do the guide after this ships. Without the anchors, the users won't be prompted.